### DISCUSSION TITLE
Postgresql native client for wsl pg_dump in windows

### DISCUSSION THREAD

**Discussion by piradata** [dbeaver/dbeaver#20627] [Q&A]
Hello guys, i want to use the backup funcionality of dbeaver on windows with the wsl binaries, but it breaks because it looks for a '.exe' file

![image](https://github.com/dbeaver/dbeaver/assets/16529503/d97a998b-393d-4429-9eae-67f9d4413645)

What can i do to just use the 'pg_dump' command instead of 'pg_dump.exe'? Dont want to install another binary only for windows...

**Comment by E1izabeth** [comment]
Have you tried to specify it with `Local client..` dialog in the left bottom corner on the provided screenshot?

**Reply by piradata** [reply]
yes, there is it:

![image](https://github.com/dbeaver/dbeaver/assets/16529503/afb6f15d-9eeb-4a0a-b58a-5a2f60b0a404)

![image](https://github.com/dbeaver/dbeaver/assets/16529503/54337e58-1550-481b-b4c8-6661960925cd)

**Reply by piradata** [reply]
the problem is that it looks for an .exe, and there is no .exe in the binary

**Reply by E1izabeth** [reply]
I see, thank you for explanation. I'm sure it's not possible.

**Comment by piradata** [comment]
humm, ok, thanks :)

I downloaded the windows binaries here and it worked, just wanted to know if there was some config that I didnt found to change the call it does to the binaries
